cvat/apps/consensus/models.py

```diff
@@ -0,0 +1,211 @@
+# Copyright (C) 2024 CVAT.ai Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from __future__ import annotations  # this allows forward references
+
+from enum import Enum
+from typing import Any, Sequence
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.forms.models import model_to_dict
+
+from cvat.apps.engine.models import Job, ShapeType, Task, User
+
+
+class ConsensusConflictType(str, Enum):
+    NoMatchingItemError = "no_matching_item"
+    FailedAttrVotingError = "failed_attribute_voting"
```

Comment:
It's possible to get failed attribute voting even if there are no attributes defined in the task.